package.json

```diff
@@ -50,6 +50,6 @@
     "protractor": "~5.1.2",
     "ts-node": "~4.1.0",
     "tslint": "^5.9.1",
-    "typescript": "^2.5.3"
+    "typescript": ">=2.4.2 <2.7.0"
```

Comment:
Tried to change this to fix travis error : 

```

5.20s$ ng test --watch=false --progress=false --code-coverage > test-log
The command "ng test --watch=false --progress=false --code-coverage > test-log" exited with 1.

```
Also tried latest version : `"typescript": "^3.3.3333"`